server/mqtt_test.go

```diff
@@ -3277,7 +3306,8 @@ func TestMQTTRetainedMsgMigration(t *testing.T) {
 	require_NoError(t, err)
 
 	// Publish some retained messages on the old "$MQTT.rmsgs" subject.
-	for i := 0; i < 100; i++ {
+	const N = 100
```

Comment:
This was an accidental change from debugging (this test was failing at one point) but it seems worth leaving